app/services/search/user.rb

```diff
@@ -29,7 +29,9 @@ def prepare_doc(hit)
           },
           "title" => hit["name"],
           "path" => hit["path"],
-          "id" => hit["id"]
+          "id" => hit["id"],
+          "class_name" => "User",
```

Comment:
Couple more fields need to render users into the articleHTML template in the main feed.